src/Lit.fs

```diff
@@ -26,3 +34,16 @@ let styles (styles: Feliz.CssStyle seq) =
     |> Seq.iter (fun (Feliz.CssStyle(key, value)) ->
         map?(key) <- value)
     Bindings.styleMap map
+    
+let classes (classes: (string * bool) seq) = 
+    Bindings.classMap (keyValueList CaseRules.LowerFirst classes)
```

Comment:
I think we don't really need lit-html `classMap` because it's trivial to do the filtering in F#.